src/pages/trade/ui/positions.tsx

```diff
@@ -309,27 +309,19 @@ const Positions = observer(({ showInactive }: { showInactive: boolean }) => {
                       <Table.Td density='slim'>
                         <div className='flex flex-col gap-4'>
                           {position.orders.map((order, i) => (
-                            <Text
+                            <ValueViewComponent
```

Comment:
I believe as per the design there's no icon in this column, but `ValueViewComponent` now supports: "showIcon={false}"